hack/k8s/helm/nuclio/templates/rolebinding/crd-admin.yaml

```diff
@@ -30,7 +31,7 @@ subjects:
 - kind: ServiceAccount
   name: {{ template "nuclio.serviceAccountName" . }}
   namespace: {{ .Release.Namespace }}
-
+{{- end }}
 {{- if eq .Values.rbac.crdAccessMode "namespaced" }}
 
 ---
```

Comment:
I think we can remove this separator, as both would never be created together after this change